.gitbook/assets/ReadonlyREST-api.yml

```diff
@@ -307,6 +386,46 @@ components:
         status: FILE_SIZE_EXCEEDS
         message: "The file size exceeds the maximum 10MB allowed limit. Please upload a smaller file."
 
+    UpdateLicenseKeySuccessResponse:
+      summary: License key updated with success
+      value:
+        status: SUCCESS
+        message: "ok"
+    UpdateLicenseKeyUnmodifiedResponse:
+      summary: License key not modified
+      value:
+        status: UNMODIFIED
+        message: "License token already exists is not modified. If you want to overwrite your license key use overwrite=true query string parameter."
+    UpdateLicenseKeyExpiredResponse:
+      summary: License key expired
+      value:
+        status: JWT_EXPIRED
```

Comment:
does it matter to the user that the License is JWT? I think JWT is an implementation detail and we should not bother users with this info

Reply:
> does it matter to the user that the License is JWT?

The user who will configure the license key will be the Admin guy who knows what's the JWT (that's my assumption). So in theory it gives additional information to the end user. I have a feeling that it's the same type of implementation detail as the cookie, which on the one side it's implementation details but on the other, it gives the user, useful information.

> I think JWT is an implementation detail and we should not bother users with this info

The problem I can see in the implementation details is that it's something that can change in the future because it was a developer's decision to implement it this way, but is it really a problem in our case?

Reply:
At the moment the license is JWT. I've just checked my license and what I see is a string (because it's JWT encoded using base64). So, from the perspective of our user, we give him some string that we call the license (through Portal), and they can upload it using ROR KBN (using API or through the UI). 

The fact that he deals with JWT should not bother them. For them, it's just a string (at least knowing that it's a JWT doesn't bring anything important for the user ... that's why, it's better to not bother them).

Then if it's really the implementation detail which is not important for users, we should not mislead them by introducing sth in the API that they could not understand.

Reply:
makes sense, changed ✔️